modules/dnn/src/op_inf_engine.hpp

```diff
@@ -101,7 +90,6 @@ CV__DNN_INLINE_NS_END
 
 // A series of wrappers for classes from OpenVINO API 2.0.
 // Need just for less conditional compilation inserts.
-#if INF_ENGINE_VER_MAJOR_GE(INF_ENGINE_RELEASE_2022_1)
 namespace InferenceEngine {
 
 class CNNNetwork {
```

Comment:
Please remove this workaround with `CNNNetwork` alias

Reply:
done